photon-ml/src/main/scala/com/linkedin/photon/ml/util/PalDBIndexMap.scala

```diff
@@ -57,14 +57,14 @@ class PalDBIndexMap extends IndexMap {
   private var _partitioner: HashPartitioner = _
 
   /**
-    * Load a storage at a particular path
-    *
-    * @param storePath The directory where the storage is put
-    * @param partitionsNum The number of partitions, the storage contains
-    * @param isLocal default: false, if set false will use SparkFiles to access cached files; otherwise,
-    *                it will directly read from local files
-    * @return
-    */
+   * Load a storage at a particular path
+   *
+   * @param storePath     The directory where the storage is put
+   * @param partitionsNum The number of partitions, the storage contains
+   * @param isLocal       default: false, if set false will use SparkFiles to access cached files; otherwise,
+   *                      it will directly read from local files
```

Comment:
We don't the consistent spacing thing right now. Whether we should or not is another question, but let's be consistent for now.